recipe/meta.yaml

```diff
@@ -16,6 +18,8 @@ build:
 
 requirements:
   build:
+    run_exports:
+      - x264 >={{ version }},<{{ next_year }}0000
```

Comment:
run_exports go under `build` not `requirements:build`.

Reply:
yup sorry, silly me... thanks